lib/Styles/Core.less

```diff
@@ -167,3 +167,15 @@ select {
   font-size: 14px;
   padding-left: 5px;
 }
+
+
+.sr-only{
```

Comment:
hi @kring sr-only means screen reader only, i think it's a pretty standard name? it's used by bootstrap as well :)